swan-lake/resources/featured-scenarios/write-a-restful-api-with-ballerina.md

```diff
@@ -28,76 +26,161 @@ This guide will walk you through creating a RESTful API with two endpoints. In t
+- The `GET` request is to retrieve data; the response should be `200 OK`.
+- The `POST` request is to add data; the response should be `201 Created`.
 
 ### The second endpoint
 
-The second endpoint is about getting data filtered from the service. The data is filtered by the ISO code. Therefore, the second endpoint accepts the ISO code as part of the URL and responds with the `200 OK` status code. In the event of an error, the relevant error is sent back to the client.
+The second endpoint is about getting data filtered from the service. The data is filtered by the ISO code. Therefore, the second endpoint accepts the ISO code as part of the URL and responds with the `200 OK` status code. The relevant error is sent back to the client in the event of an error.
 
 >**Info:** For the complete source code of this implementation, see [The complete code](/learn/write-a-restful-api-with-ballerina/#the-complete-code).
 
-## Create the service package 
+## Create the package 
 
-Ballerina uses packages to group code. You need to create a Ballerina package and write the business logic in it. In the terminal, execute the command below to create the Ballerina package for the API implementation.
+Ballerina uses packages to group code. Follow the steps below to create a Ballerina package and write the business logic. 
 
 > **Info:** For more information on Ballerina packages, see [Organize Ballerina code](/learn/organize-ballerina-code/).
 
-```
-$ bal new covid19 -t service
-```
+1. In the terminal, execute the command below to create the Ballerina package for the API implementation.
 
-You view the output below.
+    ```
+    $ bal new write-a-restful-api
+    ```
 
-```
-Created new package 'covid19' at covid19.
-```
+    You should see the output similar to the following.
 
-This creates a directory named `covid19` with the default module along with a sample code for the service as shown below. 
+    ```
+    package name is derived as 'write_a_restful_api'. Edit the Ballerina.toml to change it.
 
-```
-.
-├── covid19
-│   ├── Ballerina.toml
-│   └── service.bal
-```
+    Created new package 'write_a_restful_api' at /Users/write-a-restful-api.
+    ```
+
+    This creates a directory named `write-a-restful-api` with sample Ballerina code, as shown below. 
+
+    ```
+    .
+    ├── Ballerina.toml
+    └── main.bal
+    ```
+
+    - `Ballerina.toml` is the file that makes the folder a Ballerina package. 
+    - The `main.bal` file provides the look and feel of a simple Ballerina program. 
+
+2. Open the created package in VS Code.
 
-- `Ballerina.toml` is the file that makes the folder a Ballerina package. It also contains a test directory to include tests for the service. However, this will not be used in this guide. 
-- The `service.bal` template file provides a look and feel about Ballerina services. 
+3. Remove the generated content in the `main.bal` file and open the diagram view in VS Code.
+
```

Comment:
Extra new lines?

Reply:
<img width="882" alt="Screenshot 2024-02-21 at 09 47 57" src="https://github.com/ballerina-platform/ballerina-dev-website/assets/11707273/1df27ced-86d5-4eeb-ad73-df35a043b2df">

Reply:
This doesn't have extra new lines. @anupama-pathirage